packages/kbn-i18n/README.md

```diff
@@ -13,7 +13,7 @@ message syntax.
 
 ## Localization files
 
-Localization files have [JSON5](https://github.com/json5/json5) format.
+Localization files have JSON format but they also support [JSON5](https://github.com/json5/json5) format.
```

Comment:
Hmm, I actually tend to think that we should get rid of `json5` dependency entirely and remove all mentioning of JSON5 from this package since we're not going to support it in the near future and even if we do it's easy to return. But would still keep it in the tools.

Reason: even though we don't receive json5 files we use `json5` dep to parse them at runtime in production, I don't think we want this dependency right now, but tools are dev only so it's safe to keep.

If you agree, would you mind to create additional PR to remove json5 dep?

__nit:__ And hence `Localization files are JSON files.` would be enough.